tests/mocha/field_variable_test.js

```diff
@@ -8,54 +8,46 @@ suite('Variable Fields', function() {
       }
     };
   }
-  function fieldVariable_createAndInitField(workspace) {
-    var fieldVariable = new Blockly.FieldVariable('name1');
+  function initField(fieldVariable, workspace) {
     var mockBlock = getMockBlock(workspace);
     fieldVariable.setSourceBlock(mockBlock);
     // No view to initialize, but still need to init the model.
     fieldVariable.initModel();
     return fieldVariable;
   }
-
+  function createAndInitFieldConstructor(workspace, variableName) {
+    return initField(new Blockly.FieldVariable(variableName), workspace);
+  }
+  function createAndInitFieldJson(workspace, variableName) {
+    return initField(Blockly.FieldVariable.fromJson(
+        { variable:variableName }), workspace);
+  }
+  function assertValueDefault(variableField) {
+    var actualValue = variableField.getValue();
```

Comment:
From code, the default name will be 'i' and the default value will be a random UUID. It might be worth mocking some of the methods in Blockly.Variables so you can control the defaults and make the tests for default clearer.

Reply:
Note: It looks like sinon is the proper way to do mocking in mocha.

Example: https://github.com/google/blockly/blob/develop/tests/mocha/event_test.js#L84
Docs: https://sinonjs.org/